source/manual/incident-reports.html.md

```diff
@@ -60,30 +60,34 @@ Time to fix (time from when incident declared to marked as resolved)|
 ### Time:
 ### Attendance:
 
-### Root cause
+### Root cause analysis
 
 Use this section to summarise the root cause of the incident. A draft root cause should be included, and discussed and agreed upon during the incident review. This root cause should be written for a non-technical audience.
```

Comment:
Should this paragraph also be updated -> `A root cause is an underlying factor why an incident occurred, which when fixed would prevent a recurrence. Identify and summarise all of the root causes that led to the incident in a short paragraph, aimed at a non-technical audience.`